README.md

```diff
@@ -106,6 +108,7 @@ Options:
   --public <value>           The public hostname/ip address of the server.
   --firewall [value...]      Enable firewall or set hosts that are allowed to access the dev server.
   --no-firewall              Disable firewall.
+  --watch-files <value...>   Watch static files for file changes.
```

Comment:
```suggestion
  --watch-files [value...]   Watch static files for file changes.
```

Reply:
Value is required  https://github.com/webpack/webpack-dev-server/blob/d9a1cec198d8eb6e9fb8a0781294499f89103012/bin/cli-flags.js#L233